src/App/Fossa/Container/Sources/DockerArchive.hs

```diff
@@ -163,6 +163,7 @@ analyzeLayer systemDepsOnly filters capabilities osInfo layerFs tarball = do
           (projectResults, ()) <-
             runTarballReadFSIO layerFs tarball
               . runReader noExperimental
+              . runReader (mempty :: MavenScopeFilters)
```

Comment:
[Optional] use `noMavenScopeFiltering` and provide comment to signify why we do not care about maven scope filtering here!

Reply:
Ack will reformat. 

Question, the reason why I set MavenScopeFilters to mempty was because I was following the same notation as noExperiemental where they have the experiemental configs set empty. Is it safe to assume that mavenScopeFiltering isn't allowed when analyzing from a docker archive or did I misinterpret this?